test/examples/nodejs_postgresql_test.go

```diff
@@ -0,0 +1,540 @@
+package examples_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/redhat-developer/service-binding-operator/pkg/apis/apps/v1alpha1"
+	"github.com/redhat-developer/service-binding-operator/test/examples/util"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var (
+	exampleName  = "nodejs_postgresql"
+	operatorsNS  = "openshift-operators"
+	operatorName = "service-binding-operator"
+	appNS        = "service-binding-demo"
+
+	nodeJSApp = "https://github.com/pmacik/nodejs-rest-http-crud"
+	appName   = "nodejs-rest-http-crud"
+
+	clusterAvailable = false
+
+	pkgManifest                      = "db-operators"
+	bckSvc                           = "postgresql-operator"
+	dbName                           = "db-demo"
+	sbr                              = ""
+	podNameSBO, podNameBackingSvcOpr string
+)
+
+func TestNodeJSPostgreSQL(t *testing.T) {
+
+	t.Run("set-example-dir", SetExampleDir)
+	t.Run("get-oc-status", GetOCStatus)
+
+	t.Run("install-service-binding-operator", MakeInstallServiceBindingOperator)
+	t.Run("install-backing-service-operator", MakeInstallBackingServiceOperator)
+	t.Run("create-project", CreateProject)
+	t.Run("import-nodejs-app", ImportNodeJSApp)
+	t.Run("create-backing-db-instance", CreateBackingDbInstance)
+	t.Run("createservice-binding-request", CreateServiceBindingRequest)
+	t.Run("delete-project", DeleteProject)
+	t.Run("uninstall-backing-db-instance", UninstallBackingServiceOperator)
+	t.Run("uninstall-service-binding-operator", UninstallServiceBindingOperator)
```

Comment:
This block implies that ech `t.Run()` is a separate test, but it is not. the whole `TestNodeJSPostgresSQL` is just single test, executing the example called "Binding an Imported app to an In-cluster Operator Managed PostgreSQL Database". We should name the test function so it reflects that, but we should not have any t.Run() statements in here.

the functions that are invoked could be private. Also, if you want to fail fast, each of them could return boolean so that the test can fail using `require`.